src/Components/MultipleFields.tsx

```diff
@@ -0,0 +1,68 @@
+import { Stack, H1 } from "@deskpro/app-sdk";
+import { IAzureWorkItem, IAzureWorkItemFields } from "../types/azure/workItem";
+import { splitArrEvery2 } from "../utils/utils";
+import { workItemFields } from "../utils/workItemFields";
+import { TwoColumn } from "./TwoColumn";
+
+type Props = {
+  data: string[];
+  processName: string;
+  workItemTypeName: string;
+  item: IAzureWorkItem;
+  fieldName: string;
+};
+
+export const MultipleFields = ({
+  data,
+  processName,
+  workItemTypeName,
+  item,
+  fieldName,
+}: Props) => {
+  return (
+    <Stack vertical gap={12} style={{ width: "100%" }}>
+      <H1>{fieldName}</H1>
+      {splitArrEvery2(data).map((field, i) => {
```

Comment:
Can it be better to use `lodash` for use utilities like this? for this case use [chunk](https://lodash.com/docs/4.17.15#chunk)

Reply:
It does the exact same thing. Dont think it would be necessary to install a whole lib just to use one method

Reply:
Try to use more methods))) I trust `Lodash` more than myself. The code there is tested and more reliable

Reply:
Use the import of only one function so as not to load the whole library

`import chunk from "lodash/chunk";`